__tests__/server-get.test.js

```diff
@@ -247,3 +247,88 @@ describe("GET/api/reviews/:review_id/comments", () => {
       });
   });
 });
+
+describe("/api/reviews/:review_id/comments POST tests", () => {
+  test("Status: 202, Test that a new comment is posted", () => {
+    const data = {
+      username: "philippaclaire9",
+      body: "This was an epic journey!"
+    };
+    return request(app)
+      .post("/api/reviews/2/comments")
+      .send(data)
+      .expect(202)
+      .then(({ body }) => {
+        body.newComment.forEach((comments) => {
+          expect(comments).toEqual({
+            comment_id: 7,
+            author: "philippaclaire9",
+            review_id: 2,
+            votes: 0,
+            created_at: expect.any(String),
+            body: "This was an epic journey!"
+          });
+        });
+      });
+  });
+  test("Status: 202, Posts comments with the correct properties", () => {
+    const data = {
+      username: "philippaclaire9",
+      body: "This was an epic journey!"
+    };
+    return request(app)
+      .post("/api/reviews/2/comments")
+      .send(data)
+      .expect(202)
+      .then(({ body }) => {
+        body.newComment.forEach((comments) => {
+          expect(comments).toEqual({
+            comment_id: 7,
+            author: "philippaclaire9",
+            review_id: 2,
+            votes: 0,
+            created_at: expect.any(String),
+            body: "This was an epic journey!"
+          });
+        });
+      });
+  });
+  test("status: 404, When the comments part of the query is entered incorrectly", () => {
+    const data = {
+      username: "philippaclaire9",
+      body: "This was an epic journey!"
+    };
+    return request(app)
+      .post("/api/reviews/2/Wrongcommentsquery")
+      .send(data)
+      .expect(404)
+      .then(({ body }) => {
+        expect(body.msg).toBe("URL Not Found");
+      });
+  });
+   test("status: 400, invalid review_id format", () => {
+     return request(app)
+       .get("/api/reviews/this_is_not_an_id/comments")
+       .expect(400)
+       .then(({ body }) => {
+         expect(body.msg).toBe("Invalid input");
+       });
+   });
+   test("status: 400, No such key, wrong key, wrong body", () => {
+     return request(app)
+       .get("/api/reviews/this_is_not_an_id/comments")
+       .expect(400)
+       .then(({ body }) => {
+         expect(body.msg).toBe("Invalid input");
+       });
+   });
+   test("status: 400, No such key, wrong key, wrong userName", () => {
+     return request(app)
+       .get("/api/reviews/this_is_not_an_id/comments")
+       .expect(400)
+       .then(({ body }) => {
+         expect(body.msg).toBe("Invalid input");
+       });
+   });
```

Comment:
These last three tests are all using .get instead of .post. 

You want to attach a .send too to send the comment you wish to try and post